autoextract_poet/util.py

```diff
@@ -0,0 +1,22 @@
+from typing import Type, Optional
+from weakref import WeakKeyDictionary
+
+import attr
+
+
+# Caches the attribute names for attr.s classes
+CLASS_ATTRS: WeakKeyDictionary = WeakKeyDictionary()
+
+
+def attr_prepare(data: Optional[dict], item_cls: Type) -> dict:
```

Comment:
What do you think about calling it "remove_unknown_fields" or something like that, to describe better what kind of preparation the functionis doing?

Reply:
Makes total sense. Done in https://github.com/scrapinghub/autoextract-poet/pull/9/commits/f306ec74c896869405ede2bc41647dae53602198